### PR TITLE
net-irc/znc: fix googlecode URI, add unzip DEPEND, minor fixes

### PR DESCRIPTION
**Pull request by sbraz**
I only fixed the googlecode URI in the latest version and got rid of 1.6.1 as it's not stable anyway.

### CODE REVIEW

File: net-irc/znc/Manifest [hunk 1]
```diff
@@ -1,4 +1,3 @@
-DIST gtest-1.7.0.zip 1164254 SHA256 247ca18dd83f53deb1328be17e4b1be31514cedfc1e3424f672bf11fd7e0d60d SHA512 8859369f2dd32cbc2ac01aba029aa3ff20a321f40658b9643aff442d34c33468221866b801b28c66a28af47dbcd362d26941fc98db92b6efb7e41ea5b7be1a07 WHIRLPOOL 0c31a385159551859c1afe76480b3fb1b560d666db9a0afc5cbda92bcd53bf129f85a8f902c6ded0779c2b4c49aacec59ba5a4d5ce316a07bf08174f4fc64049
+DIST googletest-release-1.7.0.zip 633003 SHA256 b58cb7547a28b2c718d1e38aee18a3659c9e3ff52440297e965f5edffe34b6d0 SHA512 7fff71ef2226b70843683d5c5d09f09f7fc2151a44c66d1dcf9ebdbe695cbc4c0cef330798af926b3bd65d156cece55b8015f78283e06cc6696d1ceab821cbaa WHIRLPOOL 0976aa65bd63c60539cc23f8c143bc17cd2867f0c7d516962c280e16140fbcf54a3e91890fbf456beb7afc8981555aa3a065a5a032bb1c105de12786a1241066
```
Comment (mgorny) [inline net-irc/znc/Manifest]: I see different checksums. Did anything change besides directory name?

File: net-irc/znc/znc-1.6.3.ebuild [hunk 2]
```diff
@@ -9,11 +9,13 @@ inherit eutils python-single-r1 readme.gentoo-r1 systemd user
 
 MY_PV=${PV/_/-}
 GTEST_VER="1.7.0"
-GTEST_URL="https://googletest.googlecode.com/files/gtest-${GTEST_VER}.zip"
+GTEST_URL="https://github.com/google/googletest/archive/release-${GTEST_VER}.zip -> googletest-release-${GTEST_VER}.zip"
```
Comment (mgorny) [inline net-irc/znc/znc-1.6.3.ebuild]: Use `.tar.gz` instead.

**Comment by dwfreed** [comment]
That bit was reverted

File: net-irc/znc/znc-1.6.3-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+RDEPEND="
+	dev-libs/icu:=
+	sys-libs/zlib
+	perl? ( >=dev-lang/perl-5.10:= )
+	python? ( ${PYTHON_DEPS} )
+	sasl? ( >=dev-libs/cyrus-sasl-2 )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl )
```
Comment (SoapGentoo) [inline net-irc/znc/znc-1.6.3-r1.ebuild]: libressl needs `:0=` too

File: net-irc/znc/znc-1.6.3-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+		$(use_enable perl) \
+		$(use_enable python) \
+		$(use_enable sasl cyrus) \
+		$(use_enable ssl openssl) \
+		$(use_enable tcl tcl) \
+		$(use_with test gtest "${WORKDIR}/googletest-release-${GTEST_VER}")
+}
+
+src_install() {
+	emake install DESTDIR="${D%/}"
```
Comment (SoapGentoo) [inline net-irc/znc/znc-1.6.3-r1.ebuild]: `default`?

File: net-irc/znc/znc-1.6.3-r1.ebuild [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+	dodoc NOTICE README.md
```
Comment (SoapGentoo) [inline net-irc/znc/znc-1.6.3-r1.ebuild]: `dodoc NOTICE` (as README.md will have been installed by einstalldocs)

**Comment by sbraz** [comment]
I addressed all the changes you guys mentioned, thanks for the feedback!

File: net-irc/znc/znc-1.6.3-r1.ebuild [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+		$(use_enable python) \
+		$(use_enable sasl cyrus) \
+		$(use_enable ssl openssl) \
+		$(use_enable tcl tcl) \
+		$(use_with test gtest "${WORKDIR}/googletest-release-${GTEST_VER}")
+}
+
+src_install() {
+	default
+	dodoc NOTICE
```
Comment (dwfreed) [inline net-irc/znc/znc-1.6.3-r1.ebuild]: Why not put this in DOCS, so einstalldocs handles this for you?

Comment (mgorny) [inline net-irc/znc/znc-1.6.3-r1.ebuild]: @dwfreed, please avoid the ping-pong. @sbraz has just been told to switch from dodoc to default + dodoc for additional files. It doesn't really matter how it's done, and there is really no point in changing the way it's done three times a day…

**Comment by mgorny** [comment]
One little post-push remark: for minor changes like slot op addition, you can copy the stable keywords over to the new revision and remove the old one.

**Comment by mgorny** [comment]
(I've done that)

**Pull request closed without merging**